### ISSUE TITLE
Build fail with blog post

### ISSUE THREAD

**Issue by m-blc** [unixcorn-project/u_website#24]
Hello,

After creating a first blog post, the build fail. Here are the logs given by `middleman build --verbose`:
```
== Request: /blog/2018/unixcorn-3-ans/index.html
       error  build/blog/2018/unixcorn-3-ans/index.html
undefined method `display_name' for nil:NilClass
layouts/blog_layout.erb:7:in `block (2 levels) in singleton class'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/core_extensions/default_helpers.rb:13:in `capture_from_template'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/core_extensions/default_helpers.rb:68:in `capture_html'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/template_context.rb:82:in `wrap_layout'
layouts/blog_layout.erb:3:in `block in singleton class'
layouts/blog_layout.erb:-10:in `instance_eval'
layouts/blog_layout.erb:-10:in `singleton class'
layouts/blog_layout.erb:-13:in `__tilt_47175955244680'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/tilt-2.0.8/lib/tilt/template.rb:170:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/tilt-2.0.8/lib/tilt/template.rb:170:in `evaluate'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/tilt-2.0.8/lib/tilt/template.rb:109:in `render'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/file_renderer.rb:79:in `render'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_reference.rb:43:in `send_to'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/call_with.rb:76:in `call_with'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/template_renderer.rb:157:in `block in render'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/activesupport-5.0.6/lib/active_support/notifications.rb:166:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/util.rb:21:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/template_renderer.rb:156:in `render'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_reference.rb:43:in `send_to'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/call_with.rb:76:in `call_with'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/sitemap/resource.rb:154:in `render'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_reference.rb:43:in `send_to'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/call_with.rb:76:in `call_with'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-blog-4.0.2/lib/middleman-blog/blog_article.rb:64:in `render'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/rack.rb:112:in `process_request'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/rack.rb:66:in `block in call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/rack.rb:65:in `catch'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/rack.rb:65:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/urlmap.rb:68:in `block in call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/urlmap.rb:53:in `each'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/urlmap.rb:53:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/extensions/minify_javascript.rb:55:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/extensions/minify_css.rb:63:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-autoprefixer-2.9.0/lib/middleman-autoprefixer/extension.rb:48:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/head.rb:12:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/lint.rb:49:in `_call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/lint.rb:37:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/builder.rb:153:in `call'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/mock.rb:74:in `request'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rack-2.0.4/lib/rack/mock.rb:56:in `get'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/builder.rb:232:in `block in output_resource'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/activesupport-5.0.6/lib/active_support/notifications.rb:166:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/util.rb:21:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/builder.rb:225:in `output_resource'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_reference.rb:43:in `send_to'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/call_with.rb:76:in `call_with'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:484:in `call_with_index'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:455:in `process_incoming_jobs'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:437:in `block in worker'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:428:in `fork'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:428:in `worker'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:419:in `block in create_workers'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:418:in `each'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:418:in `each_with_index'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:418:in `create_workers'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:358:in `work_in_processes'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/parallel-1.12.1/lib/parallel.rb:264:in `map'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/builder.rb:137:in `output_resources'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_reference.rb:43:in `send_to'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/call_with.rb:76:in `call_with'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/builder.rb:131:in `output_files'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_reference.rb:43:in `send_to'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/call_with.rb:76:in `call_with'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/builder.rb:71:in `block in run!'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/activesupport-5.0.6/lib/active_support/notifications.rb:166:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/util.rb:21:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/builder.rb:70:in `run!'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_reference.rb:43:in `send_to'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/call_with.rb:76:in `call_with'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/contracts-0.13.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-cli-4.2.1/lib/middleman-cli/build.rb:80:in `block in build'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/activesupport-5.0.6/lib/active_support/notifications.rb:166:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-core-4.2.1/lib/middleman-core/util.rb:21:in `instrument'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-cli-4.2.1/lib/middleman-cli/build.rb:79:in `build'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `block in invoke_all'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `each'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `map'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `invoke_all'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/group.rb:232:in `dispatch'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:115:in `invoke'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:40:in `block in register'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
/home/u_website/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/middleman-cli-4.2.1/bin/middleman:70:in `<top (required)>'
/home/u_website/.rbenv/versions/2.5.0/bin/middleman:23:in `load'
/home/u_website/.rbenv/versions/2.5.0/bin/middleman:23:in `<main>'
``` 

Maybe @BaptisteGelez have you already face this issue?

Thanks!

**Comment by BaptisteGelez** [comment]
I think it is because you forgot the `author` field in the frontmatter of your article.

**Comment by m-blc** [comment]
@BaptisteGelez thanks!

Fixed by commit 945f17ee3e62366b5b8a98397162740d8388c248
